thinkhazard/templates/admin/index.jinja2

```diff
@@ -56,7 +56,7 @@
               <a
                 class="btn btn-info"
                 href="{{request.route_url('admin_add_task', _query={'task': 'process'})}}"
-                {% if 'process' in running_keys %}
+                {% if running|length > 0 %}
```

Comment:
`running_keys` is not needed anymore, could be removed in the view

Reply:
:+1: